app/src/views/AccountMyFormsDref/DrefTableActions/i18n.json

```diff
@@ -13,6 +13,8 @@
         "drefApprovalInProgressTitle": "Approval in progress...",
         "drefAccountCouldNotCreate": "Could not create new operational update",
         "drefAccountCouldNotCreateFinalReport": "Could not create final report",
-        "drefAccountConfirmMessage": "You're about to Approve this DREF. Once approved, it can no longer be edited. Are you sure, you want to Approve?"
+        "drefAccountConfirmMessage": "You're about to Approve this DREF. Once approved, it can no longer be edited. Are you sure, you want to Approve?",
```

Comment:
```suggestion
        "drefAccountConfirmMessage": "You're about to Approve this DREF. Once approved, it can no longer be edited. Are you sure you want to Approve?",
```